crates/open_ai/src/open_ai.rs

```diff
@@ -100,6 +105,7 @@ impl Model {
             Model::Four => 8192,
             Model::FourTurbo => 128000,
             Model::FourOmni => 128000,
+            Self::FourOmniMini => 128000,
```

Comment:
Self instead of `Model`?

Reply:
lmao, uh I mean, it works.... so yes? but I can go standardize that.

Reply:
Actually looking closer it should be using Self it's the ones using Model that don't fit in with the rest of the code

<img width="1402" alt="image" src="https://github.com/user-attachments/assets/771a86d4-f8e4-4026-84f5-e36772239525">

Reply:
that's fair.  might as well refactor `serialize_model` line 119 to `Self` as well if you're already at it

Reply:
That's outside of the Model impl so needs the proper Model type, I could refactor it into model that would mean changing the structure of how it gets called across the project

Reply:
I didn't notice that, got it